test/test-execute/exec-bind-paths.service

```diff
@@ -0,0 +1,14 @@
+[Service]
+Type=oneshot
+# Create a file in /tmp/test-exec_bind_paths
+ExecStart=/bin/touch /tmp/test-exec_bind_paths/thisisasimpletest
```

Comment:
I wasn't sure what could be used instead of `/bin/test`, so I ran the following script to get the most frequently used command in `test/test-execute/*`:
```sh
$ grep -r Exec test/test-execute/ | perl -lne 'print $1 if /^.*?Exec[^=]*=\+?([^ ]+)/' | sort | uniq -c | sort -rn
     79 /bin/sh
      6 /usr/bin/test
      6 /bin/echo
      4 /bin/touch
      2 /bin/rm
```
Would it better to use `/bin/sh -c` everywhere for consistency?

Reply:
OK. Updated.

Reply:
Thank you. That looks much better to me. On the other hand, I've just seen https://github.com/systemd/systemd/pull/7066/files#diff-8e6cffd3fb956dffc2835f68f0ad79cf and I'm a bit confused about why @poettering prefers `/usr/bin/test` to `/bin/sh`.